source/Mission.cpp

```diff
@@ -346,6 +360,13 @@ void Mission::Save(DataWriter &out, const string &tag) const
 			out.Write("blocked", blocked);
 		if(deadline)
 			out.Write("deadline", deadline.Day(), deadline.Month(), deadline.Year());
+		if(completeAnywhere)
+			out.Write("complete", "anywhere");
+		else if(!completionFilter.IsEmpty() && completionFilter.IsValid())
+		{
+			out.Write("complete", "at");
+			completionFilter.Save(out);
+		}
```

Comment:
Again, do this right after `destination`.

Reply:
The deed is done.